src/plugins/discover/public/application/angular/discover.html

```diff
@@ -141,7 +130,7 @@ <h1 class="euiScreenReaderOnly">{{screenTitle}}</h1>
                 on-remove-column="removeColumn"
               ></doc-table>
 
-              <a tabindex="0" id="discoverBottomMarker"></a>
+              <a tabindex="0" id="discoverBottomMarker">&#8203;</a>
```

Comment:
This symbol actually adds a line of whitespace, is this intended?

Reply:
Yes. From my tests Safari won't scroll to the marker if empty. For this reason I had to add this empty space character to the node.